pkg/encoding/v1/encoding.go

```diff
@@ -179,10 +179,55 @@ type ImageRef string
 
 // FIXME: implement ImageRef unmarshalling
 
+type Path string
```

Comment:
Hmm, I guess in this case its ok. As there is no `object.Path` its inside `Mount` struct.

But if we don't need  custom `UnmarshalYAML` for `Path` than we don't even need `Path` as separate type here.
We can use just `string` instead of `Path` in `Mount` struct